docs/getting_started/install_manually.md

```diff
@@ -503,7 +503,7 @@ First, change `www-data` to your web server user.
 ##### Clean the cache/ and logs/ directories
 
 ``` bash
-$ rm -rf var/cache/* var/logs/*
+rm -rf var/cache/* var/logs/*
```

Comment:
`rm -rf var/cache/* var/logs/* var/sessions/*`

Reply:
if so, then do it on all examples (adding sessions)